content/concepts/interoperability/cross-chain-bridges.md

```diff
@@ -0,0 +1,127 @@
+
+### Cross-Chain Transfer: XChainCreateClaimID
+
+This transaction checks out a cross-chain claim ID that is used for a cross-chain transfer. It is submitted on the destination chain, not the source chain. This is the first step of a cross-chain transfer of value. 
+
+A cross-chain claim ID essentially represents one cross-chain transfer of value. 
+
+### Cross-Chain Transfer: XChainCommit
+
+This transaction initiates a cross-chain transfer of value. This is done on the source chain, and locks/burns the value (“commits” the value), so that the equivalent amount can be minted/unlocked on the destination chain. Essentially, it tells the witness servers that the value was locked/burned. This value is tied to a specific cross-chain claim ID (which is included in the transaction).
+
+The account that owns the cross-chain claim ID on the destination chain is the account that controls the funds on the other end of the bridge. The funds go to the destination account specified in the XChainCommit transaction, if specified. If the destination account is not specified, then the claim ID owner must submit an XChainClaim transaction to determine where the funds will go on the destination chain.
+
+### Cross-Chain Transfer: XChainAddAttestation
+
+This transaction is submitted on the destination chain, by the witness server or anyone with access to the signatures from the witness server. It is a proof that an event (essentially just a locking/burning of funds) happened on the source chain.
+
+When enough witnesses have submitted their proofs on the destination chain that an event has occurred, the funds will be released to the destination account in the XChainCommit transaction, if specified. Otherwise, the claim ID owner must submit an XChainClaim transaction to determine where the funds will go on the destination chain.
+
+### Cross-Chain Transfer: XChainClaim
+
+This transaction completes a cross-chain transfer of value. It allows a user to claim the value on the destination chain - the equivalent of the value locked on the source chain. A user can only claim the value if they own the cross-chain claim ID associated with the value locked on the source chain (the Account field). The user can send the funds to anyone (the Destination field). This transaction is only needed if an OtherChainDestination is not specified in the XChainCommit transaction, or if something goes wrong with the automatic transfer of funds.
+
+### Other: XChainCreateAccountCommit
+
+This transaction creates an account on the sidechain. In order to start a cross-chain transfer, you need an account on the destination chain, which is a bit of a paradox situation - so to avoid that, this is a special transaction that can create an account on the destination chain for you.
```

Comment:
- actual emdash
- "in order to"→"to"
- "a bit of a paradox situation" feels a little casual/vague. To my understanding it's not impossible to fund an account on a sidechain without this transaction, but you would need to have someone who already has funds on the sidechain to do that for you. I tried one rewording but I'm open to better ideas on this.

```suggestion
This transaction creates an account on the sidechain. To start a cross-chain transfer, you need an account on the destination chain. Since it can be hard to create an account unless you already hold assets on the destination chain, this is a special transaction that can create an account on the destination chain for you.
```